webviz_subsurface/_datainput/eclipse_init_io/pvt_common.py

```diff
@@ -822,6 +830,46 @@ def viscosity_unit(self, latex: bool = False) -> str:
         """
         raise NotImplementedError("This method cannot be called from the base class.")
 
+    def density(
+        self, region_index: int, ratio: np.ndarray, pressure: np.ndarray
+    ) -> np.ndarray:
+        """Args:
+            region_index: Index of the requested PVT region
+            ratio: numpy.ndarray of ratio values the data is requested for
+            pressure: A numpy.ndarray of pressure values the data is requested for
```

Comment:
We could maybe shorten descriptions by removing the type? Since that is already mentioned using type hinting (and sphinx auto-generated docs should then be able to show type automatically in docs).